src/collector.ts

```diff
@@ -88,22 +91,38 @@ class NaivePyParser {
 class ReqDependencyCollector implements IDependencyCollector {
     constructor(public classes: Array<string> = ["dependencies"]) {}
 
-    async collect(contents: string): Promise<Array<IDependency>> {
+    async collect(contents: string, manifestFile: string): Promise<Array<IDependency>> {
         let parser = new NaivePyParser(contents);
         return parser.parse();
     }
 
 }
 
 class NaiveGomodParser {
-    constructor(contents: string) {
-        this.dependencies = NaiveGomodParser.parseDependencies(contents);
+    constructor(contents: string, manifestFile: string) {
+        this.dependencies = NaiveGomodParser.parseDependencies(contents, manifestFile);
     }
 
     dependencies: Array<IDependency>;
 
-    static parseDependencies(contents:string): Array<IDependency> {
+    static parseDependencies(contents:string, manifestFile: string): Array<IDependency> {
         const gomod = contents.split("\n");
+
+        const vscodeRootpath = manifestFile.replace("file://", "").replace("/go.mod", "")
+        const gopackages = execSync(
+                            `cd "${vscodeRootpath}" && go list -f '{{ join .Imports "\\n" }}' ./...`,
+                            { maxBuffer: 1024 * 1200 }).toString().split("\n");
```

Comment:
As mentioned, please use async or promise version of `exec`. 

Also use cwd attribute to change working directory instead of `cd` command.
https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback